Sources/MockoloFramework/Operations/UniqueModelGenerator.swift

```diff
@@ -52,13 +51,15 @@ private func generateUniqueModels(key: String,
     
     var processedLookup = Dictionary<String, Model>()
     processedElements.forEach { (key, val) in processedLookup[key] = val }
-    
-    let nonMethodModels = models.filter {$0.modelType != .method}
-    let methodModels = models.filter {$0.modelType == .method}
+
+    let (nonMethodModels, methodModels) = models.partitioned(by: { $0.modelType == .method })
     let orderedModels = [nonMethodModels, methodModels].flatMap {$0}
-    let x = uniqueEntities(in: orderedModels, exclude: processedLookup, fullnames: processedFullNames)
-    let unmockedUniqueEntities = x.filter {!$0.value.processed}
-    
+    let unmockedUniqueEntities = uniqueEntities(
+        in: orderedModels,
+        exclude: processedLookup,
+        fullnames: processedFullNames
+    ).filter {!$0.value.processed}
```

Comment:
[nits]
I need a space before and after the brace.

```suggestion
    ).filter { !$0.value.processed }
```

Reply:
In mockolo's style, it seems that spaces are not used here.